docs/tutorial/vald-agent-standalone-on-k8s.md

```diff
@@ -0,0 +1,373 @@
+
+    To complete get started, the kubernetes cluster is required.<br>
+    Vald will run on Cloud Service such as GKE, AWS.
+    In the sense of trying to "Get-Started", [k3d](https://k3d.io/) or [kind](https://kind.sigs.k8s.io/) are easy kubernetes tools to use.
+
+## Deploy Vald Agent Standalone on Kubernetes Cluster
+
+This chapter will show you how to deploy a standalone Vald Agent using Helm and run it on your kubernetes cluster. <br>
+This chapter uses [NGT](https://github.com/yahoojapan/ngt) as Vald Agent to perform vector insertion operation, indexing and searching operations.<br>
+
+1. Clone the repository
+
+    To use the `deployment yaml` for deploy, let's clone [`vdaas/vald`](https://github.com/vdaas/vald.git) repository.
+
+    ```bash
+    git clone https://github.com/vdaas/vald.git
+    cd vald
+    ```
+
+1. Deploy Vald Agent Standalonw using Helm
+
+    There is the [values.yaml](https://github.com/vdaas/vald/blob/master/example/helm/values-standalone-agent-ngt.yaml) to deploy standalone Vald Agent.
+    Each component can be disabled by setting the value `false` to the `[component].enabled` field.
+    This is useful for deploying standalone Vald Agent NGT pods.
+
+    ```bash
+    helm repo add vald https://vald.vdaas.org/charts
+    helm install vald-agent-ngt vald/vald --values example/helm/values-standalone-agent-ngt.yaml
+    ```
+
+1. Verify
+
+    ```bash
+    kubectl get pods
+    ```
+
+    <details><summary>Example output</summary><br>
+    If the deployment is successful, Vald Agent component should be running.
+
+    ```bash
+    NAME               READY   STATUS    RESTARTS   AGE
+    vald-agent-ngt-0   1/1     Running   0          20m
+    vald-agent-ngt-1   1/1     Running   0          20m
+    vald-agent-ngt-2   1/1     Running   0          20m
+    vald-agent-ngt-3   1/1     Running   0          20m
+    ```
+
+    </details>
+
+## Running Exapmle Code
+
+1. Port Forward
+
+    At first, port-foward is required to make request from your local environment possible.
+
+    ```bash
+    kubectl port-forward service/vald-agent-ngt 8081:8081
+    ```
+
+1. Download dataset
+
+    Download [fashion-mnist](https://github.com/zalandoresearch/fashion-mnist) that is used as dataset for indexing and search query.
+
+    ```bash
+    # move to working directory
+    cd example/client/agent
+
+    # download fashion-mnist testing dataset
+    wget http://ann-benchmarks.com/fashion-mnist-784-euclidean.hdf5
+    ```
+
+1. Run
+
+    We use [`example/client/agent/main.go`](https://github.com/vdaas/vald/blob/master/example/client/agent/main.go) to run the example.<br>
+    This example will insert and index 400 vectors into the Vald from the fashion-mnist dataset via grpc.
+    And then after waiting for indexing, it will request for searching nearest vector at 10 times.
+    You will get the 10 nearest neighbor vectors for each search query.<br>
+    Run example codes by executing below command.
```

Comment:
```suggestion
    We use [`example/client/agent/main.go`](https://github.com/vdaas/vald/blob/master/example/client/agent/main.go) to run the example.<br>
    This example will insert and index 400 vectors into the Vald from the fashion-mnist dataset via gRPC.
    And then after waiting for indexing, it will request for searching the nearest vector 10 times.
    You will get the 10 nearest neighbor vectors for each search query.<br>
    Run example codes by executing the below command.
```